sklearn/decomposition/pca.py

```diff
@@ -353,6 +353,16 @@ def fit_transform(self, X, y=None):
         -------
         X_new : array-like, shape (n_samples, n_components)
 
+        Notes
+        -----
+        This method returns an f-ordered array.
+        To get the output array in c-order,
+        do a transpose of the result i.e
```

Comment:
please wrap this paragraph to longer lines.

Reply:
You need to fill-up line up to 79 characters to be precise :)

Reply:
Thanks @glemaitre ! I'm on it